onnxruntime/contrib_ops/cuda/sparse/sparse_attention_v2/compile_sparse_attention_v2.py

```diff
@@ -136,5 +137,10 @@ def generate_triton_compile_shell_script(dtype="fp16"):
 
 
 if __name__ == "__main__":
-    for dtype in ["fp16", "bf16"]:
-        generate_triton_compile_shell_script(dtype)
+    major, minor = torch.cuda.get_device_capability()
+    print(f"echo Generate sparse attention v2 kernels for compute capability:{major}.{minor}")
+    assert major >= 7, "triton only supports compute capability >= 7.0"
```

Comment:
Should `minor` also have an assert check since this PR is for sm=75?

Reply:
In theory, triton compiler can work on sm=7.0. It is a hint to user that older GPU (sm < 7) is not officially supported by triton.